doc: add more links to Bentoml/examples (#3631)
bentoml/BentoML: docs/source/concepts/runner.rst

Runner can be created from a :ref:`bentoml.Runnable <reference/core:bentoml.Runnable>`
class. By implementing a :code:`Runnable` class, users can create Runner instances that

> runs custom logic. Here's an example (adopted from `one of our example projects
> <https://github.com/bentoml/BentoML/tree/main/examples/custom_runner/nltk_pretrained_model>`_)
> creating an NLTK runner that does sentiment analysis with a pre-trained model:

Reviewer: ```suggestion
runs custom logic. Here's an example (excerpted from :examples:`one of our example projects
<custom_runner/nltk_pretrained_model>`)
creating an NLTK runner that does sentiment analysis with a pre-trained model:
```

can we add an extlinks to our `docs/source/conf.py`:

```python
extlink = {
"examples": ("https://github.com/bentoml/BentoML/tree/main/examples/%s", "%s"),  # noqa: WPS323
}

for accessing remotely. :code:`RunnableMethod` can be configured with a signature,
which is defined same as the :ref:`concepts/model:Model Signatures`.

More examples about custom runners implementing their own :code:`Runnable` class can be found at:

> `Examples: Custom Runners <https://github.com/bentoml/BentoML/tree/main/examples/custom_runner>`_

Reviewer: same as above

        return {"res": spam_detection_runner.is_spam.run(input_text)}

Custom Runnable can be also built by extending the Runnable class generated by Model
from BentoML's model store. A full example with custom metrics to monitor the model's
performance can be found at `Examples: Custom PyTorch MNIST runner

> <https://github.com/bentoml/BentoML/tree/main/examples/custom_model_runner>`_

Reviewer: Same here.

Once each model completes, the results can be compared and logged as a post processing
step.

Another example generating text using three different language models concurrently and
then classify each generated paragraph with an classification model can be found at:

> `Examples: Inference Graph Tutorial

Reviewer: same
